### DISCUSSION TITLE
雷池添加自己代理ssl无法加载

### DISCUSSION THREAD

**Discussion by voctot** [chaitin/SafeLine#664] [Q&A]
代理后，通过域名访问一直处于加载中![image](https://github.com/chaitin/SafeLine/assets/36097179/1173bc08-9207-42a2-95d5-20b1c108b4bf)

**Comment by Lvshujun0918** [comment]
似乎不能这么代理的

**Comment by Lorna0** [comment]
看不出来什么信息。能否提供更多信息？例如网络拓扑如何、配置如何等等。
